cmd/mesh/testdata/profile-dump/output/all_off.yaml

```diff
@@ -641,7 +613,7 @@ values:
       hub: docker.io/omnition
       resources:
         limits:
-          cpu: 1
+          cpu: 1Gi
```

Comment:
Is this right? Seems like a lot of CPU :-)

Reply:
changed back to 1